vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -161,6 +161,11 @@ bool TextViewManager::UpdateProperty(
   } else if (TryUpdateTextAlignment(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextTrimming(textBlock, propertyName, propertyValue)) {
   } else if (TryUpdateTextDecorationLine(textBlock, propertyName, propertyValue)) {
+    // Temporary workaround for bug in XAML which fails to flush old TextDecorationLine render
+    // Link to Bug: https://github.com/microsoft/microsoft-ui-xaml/issues/1093#issuecomment-514282402
+    winrt::hstring text(textBlock.Text().c_str());
```

Comment:
I think you can just say `auto text = textBlock.Text();` so that you don't create copies of the string

Reply:
Ah yes that makes sense. I was following the format of [this line](https://github.com/microsoft/react-native-windows/blob/b01470476fbe07a8fd609ab7c838194671232d9c/vnext/Microsoft.ReactNative/Views/TextViewManager.cpp#L51) in the AddView function, but `auto text = textBlock.Text()` is much cleaner. Want me to create a new PR for this change?